core/src/main/resources/hudson/diagnosis/Messages_de.properties

```diff
@@ -25,3 +25,6 @@ MemoryUsageMonitor.TOTAL=Total
 OldDataMonitor.DisplayName=Veraltete Daten verwalten
 OldDataMonitor.Description=Konfiguration bereinigen, um \u00DCberbleibsel alter Plugins und fr\u00FCherer Versionen zu entfernen.
 HudsonHomeDiskUsageMonitor.DisplayName=Speicherplatz-Monitor
+NullIdDescriptorMonitor.DisplayName=Fehlende Descriptor-ID
+ReverseProxySetupMonitor.DisplayName=Reverse-Proxy-Konfiguration
+TooManyJobsButNoView.DisplayName=Zuviele Elemente nicht in Ansichten organisiert
```

Comment:
Yep. We also need to go through the existing English translations to update them.